migrations/mysql/20170207193805_initial_schema/up.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE users (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  name TEXT NOT NULL,
+  hair_color TEXT
+) CHARACTER SET utf8mb4;
+
+CREATE TABLE posts (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  user_id INTEGER NOT NULL,
+  title TEXT NOT NULL,
+  body TEXT
+) CHARACTER SET utf8mb4;
+
+CREATE TABLE comments (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  post_id INTEGER NOT NULL,
+  text TEXT NOT NULL
+) CHARACTER SET utf8mb4;
+
+CREATE TABLE special_posts (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  user_id INTEGER NOT NULL,
+  title TEXT NOT NULL
```

Comment:
Make that a MEDIUMTEXT and add a test which adds exactly 64k+1 bytes of text :trollface: